src/adapter.ts

```diff
@@ -83,8 +115,8 @@ class Adapter {
    *
    * Called to indicate that a device is now being managed by this adapter.
    */
-  handleDeviceAdded(device) {
-    this.devices[device.id] = device;
+  handleDeviceAdded(device: Device) {
```

Comment:
I do not have strong opinions on this but some prefer to add `: void` as return type.

Reply:
No problem.
Would you prefer explicit return types on every function?

Reply:
That's how I would code my personal stuff but I am super new to this project so I simply don't know all rules here :-) @mrstegeman what do you think?

Reply:
`@typescript-eslint/recommended` spits out warnings for every function without return type, so I guess it's a good idea.